packages/ceramic-doctype-three-id/src/__tests__/three-id-doctype.test.ts

```diff
@@ -118,36 +165,53 @@ describe('ThreeIdHandler', () => {
 
     await expect(ThreeIdDoctype._makeRecord(doctype, null, RECORDS.r1.desiredContent)).rejects.toThrow(/No DID/)
     const record = await ThreeIdDoctype._makeRecord(doctype, user, RECORDS.r1.desiredContent)
-    expect(record).toEqual(RECORDS.r1.record)
+    expect(record).not.toBeNull()
```

Comment:
`.toBeDefined()` maybe? `.not.toBeNull()` will compare with an explicit `null` but not `undefined`, is that the expected behavior?

Reply:
makes sense! fixed it